src/ServiceStack/RequestLogsFeature.cs

```diff
@@ -96,7 +96,7 @@ public void Register(IAppHost appHost)
             }
 
             appHost.GetPlugin<MetadataFeature>()
-                .AddDebugLink(AtRestPath, "Request Logs");
+                .AddDebugLink(AtRestPath.TrimStart('/'), "Request Logs");
         }
```

Comment:
I think I should have added just a bit more to that comment. This issue is only present when debugging on a local IIS server and the website is under a virtual directory.
In my case, I have a DemoApp project with a URL of _http://localhost/DemoApp/_.
The metadata and everything else comes up correctly at _http://localhost/DemoApp/*_
However, the request logs link is _http://localhost/requestlogs_ when it should be _http://localhost/DemoApp/requestlogs_